tree.go

```diff
@@ -226,9 +224,9 @@ walk:
 			}
 
 			// Otherwise insert it
-			if c != ':' && c != '*' {
+			if path[0] != ':' && path[0] != '*' {
 				// []byte for proper unicode char conversion, see #65
-				n.indices += bytesconv.BytesToString([]byte{c})
+				n.indices += path[0:1]
```

Comment:
here maybe we should add it, thanks!

Reply:
Writing like this will be more clear and concise！